VMEncryption/main/check_util.py

```diff
@@ -326,3 +328,33 @@ def is_non_fatal_precheck_failure(self):
             detected = True
             self.logger.log("PRECHECK: Unsupported mount scheme detected")
         return detected
+
+    def preInitializationCheck(self,logger,imdsStoredResults,iMDSUtil):
```

Comment:
Please use the _ naming convention followed elsewhere in the file. pre_initialization_check

Reply:
done